src/styles/parts/card.scss

```diff
@@ -0,0 +1,94 @@
+.card {
+  width: 200px;
+  height: 400px;
+  display: flex;
+  box-sizing: border-box;
+  flex-direction: column;
+  align-items: center;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__image {
+    height: 134px;
+    width: 160px;
+    margin-bottom: 40px;
+  }
+
+  // &__info{
+  //   height: 266px;
+  //   width: 160px;
+  // }
+
+  &__title {
+    font-weight: 500px;
+    font-size: 12px;
+    line-height: 18px;
+    margin: 0;
+  }
+
+  &__code {
+    color: #616070;
+    font-weight: 400;
+    font-size: 10;
```

Comment:
The font-size value is missing a unit. It should be specified in 'px', 'em', etc.